ontology/catplus_ontology.ttl

```diff
@@ -865,7 +867,6 @@ cat:Batch a rdfs:Class, sh:NodeShape ;
 cat:Plate a rdfs:Class, sh:NodeShape ;
     skos:prefLabel "Plate" ;
     skos:definition "A plate is a flat, rigid structure that holds samples or reagents in an experiment." ;
-    sh:property [sh:path cat:hasWell ; sh:class cat:Well] ; #hasWell

```

Comment:
I have removed the `hasWell` here as it aligns better with the data to have a `hasPlate` on the `cat:Well`, see below.